lib/std/os/linux.zig

```diff
@@ -1878,6 +1878,17 @@ pub fn recvmsg(fd: i32, msg: *msghdr, flags: u32) usize {
     }
 }
 
+pub fn recvmmsg(fd: i32, msgvec: [*]mmsghdr, vlen: u32, flags: u32, timeout: *timespec) usize {
```

Comment:
* Should `mmsghdr` be `?[*]mmsghdr` for the case where `vlen = 0`?
* `timeout` should indeed be `?*timespec`.

Reply:
I made this change.

Question: would it be better to take a slice of mmsghdr? and then calculate ``vlen`` as ``msgvec.len``?

I did confirm this works on 64-bit, I have not tested on 32-bit, but I didn't see anything in glibc that indicated it should be ``socketcall``.

Reply:
> Question: would it be better to take a slice of mmsghdr? and then calculate `vlen` as `msgvec.len`?

The vast majority of syscall wrappers in this file use pointers. `process_vm_readv` and `process_vm_writev` look like the only (odd) exceptions. So let's leave it like this for now.